gapic/schema/metadata.py

```diff
@@ -188,6 +192,29 @@ def python_import(self) -> imp.Import:
                 alias=self.module_alias,
             )
 
+        if self.is_proto_plus_type:
+            # We need to change the import statement to use an
+            # underscore between the module and the version. For example,
+            # change google.cloud.documentai.v1 to google.cloud.documentai_v1.
+            # Check if the package name contains a version.
+            version_regex = "^v\d[^/]*$"
+            regex_match = re.match(version_regex, self.package[-1])
+
+            if regex_match:
+                versioned_module = f"{self.package[-2]}_{regex_match[0]}"
```

Comment:
Can a package have less than 2 elements? (so [-2] raise IndexError)

Reply:
Good catch! Fixed in https://github.com/googleapis/gapic-generator-python/pull/1596/commits/3f95c0af570cd68db0dc634f79a5760573d3f406